pages/api/utils/get_paginated_data.tsx

```diff
@@ -12,14 +12,21 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
   const skipCount = parseInt((req.query.skip).toString())
   const resultSource = (req.query.resultSource).toString()
 
+  const order = (req.query.orderBy).toString()
+  const sortOrder = (req.query.sortOrder).toString() === 'desc' ? 'desc' : 'asc'
+
   let query:any = prisma.bookings
   if (resultSource === 'prospects') {
     query = prisma.prospects
   }
+  
+  let orderBy:any = { id: sortOrder }
```

Comment:
is there a way to dirrectly use what we get from the request?

Reply:
the first time when we call the request, we will not have data (some parameters depending on which to display the list), because there must be something by default, to display the list even if we have not selected according to what to display. So we cannot use what we the request directly.

Reply:
we can use the ?? operator if we have nothing in the variable (works similar to // in perl)

Reply:
my mistake. we can't do that because we have to write the query manually, we can't take it from a string. practically we cannot replace "id" or "startsat" with a string that has the value "id" or "startsat"